### PR TITLE
feat(webhosting): add connect via FTP doc MTA-1798

### PR DESCRIPTION
**Pull request by RoRoJ**

### CODE REVIEW

File: webhosting/classic/how-to/connect-via-ftp.mdx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+    <Lightbox src="scaleway-download-filezilla.png" alt="" />
+
+    The FileZilla website detects your operating system, and suggests the correct version of the FileZilla client to download.
+
+3. Click **Download FileZilla Client** and follow the on-screen prompts to complete installation.
+
+<Message type="tip">
+
+Depending on your operating system, you may prefer to download FileZilla from the command line of your terminal. On Linux Ubuntu, use the command `sudo apt-get update && sudo apt-get install filezilla`.
```
Comment (bene2k1) [inline webhosting/classic/how-to/connect-via-ftp.mdx:40]:
```suggestion
Depending on your operating system, you may prefer to download FileZilla from the command line of your terminal. On Ubuntu Linux, use the command `sudo apt update && sudo apt install filezilla`.
```

File: webhosting/classic/how-to/connect-via-ftp.mdx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+content:
+  h1: How to connect via FTP
+  paragraph: This page explains how to connect to your website via FTP using FileZilla
+tags: Filezilla FileZilla FTP
+dates:
+  validation: 2022-02-24
+  posted: 2022-02-24
+---
+
+**F**ile **T**ransfer **P**rotcol is a standard way to transfer files between a client (such as your local computer) and a server (such as your Scaleway webhosting server) via the internet. This documentation shows you how to configure the popular and free FTP client [FileZilla](https://filezilla-project.org/) to connect to your webhosting site using FTP. You will then be able to transfer files from your local computer to your Scaleway webhosting server, following the [documentation provided by Filezilla](https://wiki.filezilla-project.org/FileZilla_Client_Tutorial_(en)) if necessary.
```
Comment (ldecarvalho-doc) [inline webhosting/classic/how-to/connect-via-ftp.mdx:14]:
```suggestion
**F**ile **T**ransfer **P**rotcol is a standard way to transfer files between a client (such as your local computer) and a server (such as your Scaleway webhosting server) via the Internet. This documentation shows you how to configure the popular and free FTP client [FileZilla](https://filezilla-project.org/) to connect to your webhosting site using FTP. You will then be able to transfer files from your local computer to your Scaleway webhosting server, following the [documentation provided by Filezilla](https://wiki.filezilla-project.org/FileZilla_Client_Tutorial_(en)) if necessary.
```

File: webhosting/classic/how-to/create-ftp-account.mdx [hunk 3]
```diff
@@ -6,36 +6,11 @@ content:
...
-
-  - You have a host
-  - You have a username
-  - You have a password
-
-</Message>
-
-## How can I create an account 
-
+**F**ile **T**ransfer **P**rotcol is a standard way to transfer files between a client (such as your local computer) and a server (such as your Scaleway webhosting server) via the internet. In order to [use FTP ](/webhosting/classic/how-to/connect-via-ftp/), you must first create an account.
```
Comment (ldecarvalho-doc) [inline webhosting/classic/how-to/create-ftp-account.mdx:13]:
```suggestion
**F**ile **T**ransfer **P**rotcol is a standard way to transfer files between a client (such as your local computer) and a server (such as your Scaleway webhosting server) via the internet. In order to [use FTP](/webhosting/classic/how-to/connect-via-ftp/), you must first create an account.
```

File: webhosting/classic/how-to/create-ftp-account.mdx [hunk 4]
```diff
@@ -44,34 +19,38 @@ The File Transfer Protocol was developed to transfer files on the internet.
...
+
+  <Lightbox src="scaleway_ftpaccounts.png" alt="" />
 
-    * Your username 
-    * Your password 
-    * The path which has to be an already existing directory on your FTP
+3. Fill in the form to create your FTP account, entering the following values: 
 
-    <Lightbox src="scaleway_ftpaccounts.png" alt="" />
+    * A **username** (eg `webmaster`)
```
Comment (ldecarvalho-doc) [inline webhosting/classic/how-to/create-ftp-account.mdx:36]:
```suggestion
    * a **username** (eg `webmaster`)
```

Comment (ldecarvalho-doc) [reply webhosting/classic/how-to/create-ftp-account.mdx:36]: same for the others

File: webhosting/classic/how-to/create-ftp-account.mdx [hunk 5]
```diff
@@ -44,34 +19,38 @@ The File Transfer Protocol was developed to transfer files on the internet.
...
+    * A **password** of your choice
```
Comment (RoRoJ) [inline webhosting/classic/how-to/create-ftp-account.mdx:37]:
```suggestion
    * a **password** of your choice
```

File: webhosting/classic/how-to/create-ftp-account.mdx [hunk 6]
```diff
@@ -44,34 +19,38 @@ The File Transfer Protocol was developed to transfer files on the internet.
...
+    * A **path**, which has to be an existing directory on your FTP. Enter `/` to create an account with access to the whole directory, or a specific path to limit access to certain directories.
```
Comment (RoRoJ) [inline webhosting/classic/how-to/create-ftp-account.mdx:38]:
```suggestion
    * a **path**, which has to be an existing directory on your FTP. Enter `/` to create an account with access to the whole directory, or a specific path to limit access to certain directories.
```

**Review by bene2k1 — APPROVED** [review]

**Review by ldecarvalho-doc — APPROVED** [review]

**Pull request merged**